package.json

```
@@ -21,21 +23,31 @@
   },
   "homepage": "https://github.com/Kava-Labs/connector-config#readme",
   "dependencies": {
+    "@kava-labs/crypto-rate-utils": "^1.0.0",
+    "@kava-labs/ilp-backend-crypto": "^1.1.0",
     "bignumber.js": "^7.2.1",
+    "chokidar": "^2.0.4",
     "dotenv": "^6.1.0",
     "envkey": "^1.2.2",
-    "ilp-connector": "^22.1.0",
-    "ilp-plugin-ethereum": "next",
-    "ilp-plugin-lightning": "next",
-    "ilp-plugin-mini-accounts": "^3.5.1",
+    "execa": "^1.0.0",
+    "ilp-connector": "^22.1.3",
+    "ilp-plugin-ethereum": "^3.0.0-beta.8",
+    "ilp-plugin-lightning": "^1.0.0-beta.14",
```

```suggestion
    "ilp-plugin-lightning": "^1.0.0-beta.15",
```
Updated to use patched mini-accounts